### PR TITLE
swift 2.2-SNAPSHOT-2016-01-25-a

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

File: Library/Formula/swift.rb [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
     end
 
     resource "llvm" do
       url "https://github.com/apple/swift-llvm/archive/#{swift_tag}.tar.gz"
-      sha256 "4730fb75898110ed892d4cc35f2f58b457879c51283b19cccf797c443b3bc05e"
+      sha256 "7b07af901b6fe42793e4f25a470db8d61c60c16b92e434de17fc151861e00172"
+    end
+
+    resource "swiftpm" do
+      url "https://github.com/apple/swift-package-manager/archive/swift-DEVELOPMENT-SNAPSHOT-2016-01-25-a.tar.gz"
```
Comment (bfontaine) [inline Library/Formula/swift.rb]: Shouldn’t we use `SNAPSHOT`s instead of `DEVELOPMENT-SNAPSHOT`s here?

Comment (kashif) [inline Library/Formula/swift.rb]: actualy swift build was included later after the official 2.2 snapshot, and this development snapshot needs a patch to be able to compile, so i will remove swift package manager for the time being which will make the build pass

**Comment by mikemcquaid** [comment]
@asaph I think we should maybe wait here until we can get all the versions back in sync.

**Comment by asaph** [comment]
@mikemcquaid Did you mean @kashif?

**Comment by mikemcquaid** [comment]
@asaph Sorry, yes!

**Comment by asaph** [comment]
@mikemcquaid No worries. :smile:

**Comment by kashif** [comment]
ok closing

**Pull request closed without merging**